Dockerfile

```diff
@@ -25,9 +20,14 @@ RUN make dm-worker dm-master dmctl
 
 FROM alpine:3.10
 
+# keep compatibility
 COPY --from=builder /go/src/github.com/pingcap/dm/bin/dm-worker /dm-worker
 COPY --from=builder /go/src/github.com/pingcap/dm/bin/dm-master /dm-master
 COPY --from=builder /go/src/github.com/pingcap/dm/bin/dmctl /dmctl
-COPY --from=builder /opt/tools/bin/mydumper /mydumper
 
-EXPOSE 8291 8261 8262
+COPY --from=builder /go/src/github.com/pingcap/dm/bin/dm-worker /bin/dm-worker
+COPY --from=builder /go/src/github.com/pingcap/dm/bin/dm-master /bin/dm-master
+COPY --from=builder /go/src/github.com/pingcap/dm/bin/dmctl /bin/dmctl
+
```

Comment:
Why change binary dir

Reply:
dm-operator or users may use origin binary dir in script.

Reply:
origin binary is keeped, i just make another copy to /bin/ so it’s easier for users to use when exec pod